sys/board_software_reset/init_reset_btn.c

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2020 Benjamin Valentin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @@defgroup   module_board_software_reset board_software_reset
```

Comment:
```suggestion
 * @defgroup  board_software_reset Board software reset
 * @ingroup    sys
```

Should fix the not showing up documentation